src/components/PicklistOption/checkmark.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import StyledCheckmarkIcon from './styled/checkmarkIcon';
+
+export default function CheckmarkIcon() {
+    return (
+        <StyledCheckmarkIcon width="16px" height="10px" viewBox="0 0 16 10">
+            <path
+                d="M15.4027385,0.631586322 C15.0862366,-0.159933677 14.360892,-0.02500765 13.7424583,0.0992984521 C13.3250984,0.183659273 11.4687929,0.710638455 8.53166581,3.71430416 C7.31280819,4.96089954 6.50876321,5.95472783 5.97938121,6.71266118 C5.65599644,6.3275974 5.28551575,5.91562225 4.89644599,5.53812729 C3.69991693,4.37799518 2.36556863,3.58127161 2.30947625,3.54779009 C1.74952796,3.21581318 1.01853856,3.38663727 0.675264032,3.92927971 C0.332423067,4.47192215 0.508016606,5.18202476 1.06709776,5.51526314 C1.07853302,5.52225379 2.22438925,6.20933456 3.21427073,7.16962877 C4.22377099,8.14879246 5.13842907,9.47296826 5.14742552,9.4857932 C5.37011498,9.81051666 5.74400999,10 6.1373612,10 C6.20488884,10 6.27306682,9.99448107 6.34091963,9.98307527 C6.8051586,9.90486412 7.17807809,9.56820926 7.29178032,9.12464479 C7.29465269,9.11397486 7.76631644,7.8497189 10.257902,5.30202211 C12.2647086,3.24934726 13.6030132,2.59695692 14.0667102,2.4159885 C14.0711542,2.41525265 14.0743517,2.41493728 14.0803674,2.41372837 C14.0781454,2.41425398 14.0953796,2.40652748 14.1233445,2.39433326 C14.2029577,2.36537201 14.2456096,2.35428158 14.2480484,2.35370341 C14.2251779,2.35922234 14.2133091,2.36058894 14.2133091,2.36058894 L14.2125504,2.35659428 C14.4271105,2.26582099 14.8288078,2.09820313 14.8375875,2.09462897 C15.4424097,1.85363559 15.6390853,1.22269024 15.4027385,0.631586322 Z"
+                fillRule="nonzero"
+            />
+        </StyledCheckmarkIcon>
+    );
+}
```

Comment:
here what we do is make the icon component with its markup directly (using svg and not a styled) and then pass a style and className prop to the svg element like:
```
export default function CheckmarkIcon(props) {
    const { className, style } = props;
    return (
        <svg className={className} style={style} width="16px" height="10px" viewBox="0 0 16 10">
            <path
                d="M15.4027385,0.631586322 C15.0862366,-0.159933677 14.360892,-0.02500765 13.7424583,0.0992984521 C13.3250984,0.183659273 11.4687929,0.710638455 8.53166581,3.71430416 C7.31280819,4.96089954 6.50876321,5.95472783 5.97938121,6.71266118 C5.65599644,6.3275974 5.28551575,5.91562225 4.89644599,5.53812729 C3.69991693,4.37799518 2.36556863,3.58127161 2.30947625,3.54779009 C1.74952796,3.21581318 1.01853856,3.38663727 0.675264032,3.92927971 C0.332423067,4.47192215 0.508016606,5.18202476 1.06709776,5.51526314 C1.07853302,5.52225379 2.22438925,6.20933456 3.21427073,7.16962877 C4.22377099,8.14879246 5.13842907,9.47296826 5.14742552,9.4857932 C5.37011498,9.81051666 5.74400999,10 6.1373612,10 C6.20488884,10 6.27306682,9.99448107 6.34091963,9.98307527 C6.8051586,9.90486412 7.17807809,9.56820926 7.29178032,9.12464479 C7.29465269,9.11397486 7.76631644,7.8497189 10.257902,5.30202211 C12.2647086,3.24934726 13.6030132,2.59695692 14.0667102,2.4159885 C14.0711542,2.41525265 14.0743517,2.41493728 14.0803674,2.41372837 C14.0781454,2.41425398 14.0953796,2.40652748 14.1233445,2.39433326 C14.2029577,2.36537201 14.2456096,2.35428158 14.2480484,2.35370341 C14.2251779,2.35922234 14.2133091,2.36058894 14.2133091,2.36058894 L14.2125504,2.35659428 C14.4271105,2.26582099 14.8288078,2.09820313 14.8375875,2.09462897 C15.4424097,1.85363559 15.6390853,1.22269024 15.4027385,0.631586322 Z"
                fillRule="nonzero"
            />
        </svg>
    );
}
```
and when we pass the icon throught the styled the svg will get the styles thanks to the className passed to the svg element:
```
const StyledCheckmarkIcon = attachThemeAttrs(styled(CheckmarkIcon))`
    width: 0.8rem;
    height: 0.8rem;
    line-height: 1;
    margin-left: 0.75rem;
    flex-shrink: 0;
    fill: ${props => props.palette.brand.main};
`;
```